### DISCUSSION TITLE
Unnecessary whitespace appearing when customising template

### DISCUSSION THREAD

**Discussion by hcphoon01** [tailwindlabs/tailwindcss#4923] [Help]
I'm trying to use [this admin starter template][1] I found on the tailwind toolbox and I am trying to customise it to my liking. I've created a [codepen ](https://codepen.io/hcphoon01/pen/OJmXrRN) with the code to assist with debugging. As seen in the pen and the gif below, I get a scrollbar with lots of extra whitespace at the bottom of the page, even when there is next to no content on the page [![2]][2]

  [1]: https://www.tailwindtoolbox.com/templates/admin-template
  [2]: https://i.stack.imgur.com/38ocd.gif

**Comment by olets** [comment]
There's a ton in your sample code that doesn't affect the question. Can you reduce it to a more targeted example? Stackoverflow has a good primer on that: [How to create a Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) A good place to make a Tailwind CSS fiddle is https://play.tailwindcss.com/ (though for a general css question like this codepen will work fine too).

Looks like the vertical overflow of your main pane is exactly the height of the blue bar plus the above it.
